src/main/java/org/folio/tlib/postgres/impl/TenantPgPoolImpl.java

```diff
@@ -169,6 +172,13 @@ private String subst(String s) {
     return pgPool.preparedQuery(e);
   }
 
+  @Override
+  public PreparedQuery<RowSet<Row>> preparedQuery(String s, PrepareOptions prepareOptions) {
+    String e = subst(s);
+    log.info("preparedQuery {}", e);
+    return pgPool.preparedQuery(e, prepareOptions);
+  }
+
```

Comment:
This duplicates the code of preparedQuery(String).
To parallel https://github.com/eclipse-vertx/vertx-sql-client/blob/4.2.1/vertx-sql-client/src/main/java/io/vertx/sqlclient/impl/SqlConnectionBase.java#L75-L81
preparedQuery(String) can be changed to
```
  @Override
  public PreparedQuery<RowSet<Row>> preparedQuery(String s) {
    return preparedQuery(s, null);
  }
```

Reply:
ok

Reply:
ok